scripts/gen_template_zt.py

```
@@ -0,0 +1,681 @@
+#!/usr/bin/python3
```

Ruff (0.14.5)

Shebang is present but file is not executable

(EXE001)